spec/controllers/api/v1/reviews_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ReviewsController, type: :controller do
+  describe "POST#create" do
+    let!(:game1) { FactoryBot.create(:game) }
+    let!(:game2) { FactoryBot.create(:game) }
+
+    let!(:user1) { User.create(email: "abc@abc.com", password: "abc123") }
+    let!(:user2) { User.create(email: "123@123.com", password: "123abc") }
+
+    let!(:good_review_data) { { review: { rating: 1, body: "It's a review", user_id: user1["id"], game_id: game1["id"] } } }
+    let!(:bad_review_data) { { review: { rating: "", body: "It's another review", user_id: user2["id"], game_id: game2["id"]  } } }
```

Comment:
the syntax here is somewhat off. 
```
{ review: { rating: 1, body: "It's a review", user_id: user1["id"], game_id: game1["id"] } }
```
should be 
```
{ review: { rating: 1, body: "It's a review",  game_id: game1.id } }
```
further, the user id should not be in your good_review or bad review data params. This is automatically added based on the logic you already have in your reviews controller. remove the user_id from these hashes.